iai_semantic_maps/owl/kitchen.owl

```diff
@@ -2030,6 +2040,13 @@
         <knowrob:startTime rdf:resource="&iai-map;timepoint_0000000001"/>
         <knowrob:objectActedOn rdf:resource="&iai-map;iai_kitchen_kitchen_island_counter_top"/>
     </owl:NamedIndividual>
+    
+    <owl:NamedIndividual rdf:about="&knowrob;SemanticMapPerception_Island_MealTable">
```

Comment:
Could you please add a CT suffix to this? Because it's the coordinate of the counter top not the table itself?